modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js

```diff
@@ -157,6 +157,9 @@ const Popover = ({tooltip}) => {
 	);
 };
 
+const FIELDSET_REGEX = /Fieldset\d+/g;
+const FIELDSET_REPEAT_NUMBERS_REGEX = /\$(\d+)(?:#|\$|$)/g;
```

Comment:
i think you forgot to change this one @danvb112, sorry i didn't explicitly point it out before!

Reply:
i mean that `FIELDSET_REPEAT_NUMBERS_REGEX` should be `FIELDSET_REPEAT_INDEX_REGEX`, or you left like that intentionally?

Reply:
I forgot @rebsilva, thanks to see this!